<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="EC2AMAZ-S11DBRH" failures="1" tests="5" name="Default test" time="19.557" errors="0" timestamp="2023-07-27T05:16:53 GMT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.754">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0
at java.base/java.util.Collections$EmptyList.get(Collections.java:4553)
at StepDef.LoginStepdef.user_enters_login_credentials(LoginStepdef.java:59)
at âœ½.User enters login credentials(file:///C:/Users/Administrator/git/NEW25/NEWSEL/src/test/resources/features/Login.feature:30)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.645"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="11.605"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="16.88"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="17.057"/>
</testsuite> <!-- Default test -->
